package.json

```diff
@@ -68,8 +69,11 @@
     "@types/uuid": "^9.0.8",
     "@typescript-eslint/eslint-plugin": "^4.8.2",
     "@typescript-eslint/parser": "^4.8.2",
+    "assert": "^2.1.0",
```

Comment:
looking at the new code I don't understand the need for these new libraries, does it depend on header component of mui-italia?

Reply:
Good point, some autoimport did it for me. Thank you

Reply:
restored